closed/OpenJ9.gmk

```diff
@@ -324,14 +325,22 @@ run-preprocessors-j9 : stage-j9
 			UMA_OPTIONS_EXTRA="-buildDate $(shell date +'%Y%m%d')" \
 			tools \
 	)
+ifneq (true,$(OPENJ9_ENABLE_CMAKE))
 	@$(SED) -i -e 's/^\s*VERSION_MAJOR\s*.*=.*/VERSION_MAJOR:=8/' $(OUTPUT_ROOT)/vm/makelib/mkconstants.mk
+endif
 
 build-j9 : run-preprocessors-j9
 	@$(ECHO) Compiling OpenJ9 in $(OUTPUT_ROOT)/vm
+ifeq (true,$(OPENJ9_ENABLE_CMAKE))
+	(export OPENJ9_BUILD=true $(EXPORT_MSVS_ENV_VARS) \
+		&& $(MAKE) -C  $(OUTPUT_ROOT)/vm/build $(MAKEFLAGS) install \
```

Comment:
How does cmake set VERSION_MAJOR=8 ?

Reply:
It gets set further up,  when we call `buildtools.mk`, which will [pass it on to cmake](https://github.com/eclipse/openj9/blob/82661a78819cb7de3b80fabd69158d396a62f980/runtime/buildtools.mk#L53)